easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0209-GCCcore-12.3.0.eb

```
@@ -0,0 +1,65 @@
+# updated toolchain, version, and dependency versions
+# Thomas Eylenbosch 5-Jun-23
+
+easyblock = 'Bundle'
+
+name = 'XML-LibXML'
+version = '2.0209'
+
+homepage = 'https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod'
+description = "Perl binding for libxml2"
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkgconf', '1.9.5'),
+]
+
+dependencies = [
+    ('Perl', '5.36.1'),
+    ('Perl-bundle-CPAN', '5.36.1'),
+    ('libxml2', '2.11.4'),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
+
+exts_list = [
+    ('File::chdir', '0.1011', {
+        'source_tmpl': 'File-chdir-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN'],
+        'checksums': ['31ebf912df48d5d681def74b9880d78b1f3aca4351a0ed1fe3570b8e03af6c79'],
+    }),
+    ('Alien::Base', '2.80', {
+        'source_tmpl': 'Alien-Build-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PL/PLICEASE/'],
+        'checksums': ['d9edc936b06705bb5cb5ee5a2ea8bcf6111a3e8815914f177e15e3c0fed301f3'],
+    }),
+    ('Alien::Build::Plugin::Download::GitLab', '0.01', {
+        'source_tmpl': 'Alien-Build-Plugin-Download-GitLab-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PL/PLICEASE'],
+        'checksums': ['c1f089c8ea152a789909d48a83dbfcf2626f773daf30431c8622582b26aba902'],
+    }),
+    ('Alien::Libxml2', '0.19', {
+        'source_tmpl': 'Alien-Libxml2-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PL/PLICEASE'],
+        'checksums': ['f4a674099bbd5747c0c3b75ead841f3b244935d9ef42ba35368024bd611174c9'],
+    }),
+    ('XML::LibXML', version, {
+        'source_tmpl': 'XML-LibXML-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/'],
+        'checksums': ['b4a5abbcd689aa2fbbc8b7b45339e961c4984e48108494eb6c282b4748222425'],
+    }),
+]
+
+modextrapaths = {
+    'PERL5LIB': 'lib/perl5/site_perl/%(perlver)s/',
+}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/perl5/site_perl/%(perlver)s/x86_64-linux-thread-multi/XML/LibXML'],
```

@smoors Please use `%(arch)s` here (so it also works on Arm), and apply the same fix to the existing `XML-LibXML-2.0208-GCCcore-12.2.0.eb` where you probably inherited this from (in this PR is fine)

```suggestion
    'dirs': ['lib/perl5/site_perl/%(perlver)s/%(arch)s-linux-thread-multi/XML/LibXML'],
```

We already have this in all other `XML-LibXML` easyconfigs